### ISSUE TITLE
Datepicker component partially breaks when used with AlpineJs in Laravel 9.x app

### ISSUE THREAD

**Issue by gkimpson** [mkocansey/bladewind#17]
I have a Laravel 9.x app that uses AlpineJs in app.js (latest version of AlpineJs 3.x) - when I use the datepicker component it duplicates the days row and the selected date doesn't always work. This only happens when AlpineJs is integrated and the Alpine.start() line in resources/js/app.js is active - when I comment the line out everything seems to work perfectly fine. Any ideas? (See screenshot below)

<img width="276" alt="image" src="https://user-images.githubusercontent.com/850218/170640789-c01bb9f7-cf49-42f4-85a1-36f4c64f144d.png">

```
require('./bootstrap');
import Alpine from 'alpinejs';
window.Alpine = Alpine;

Alpine.start(); // when the line is commented out the datepicker seems to work as expected

```

Console log errors
```
alpinejs:5 Uncaught ReferenceError: day is not defined
    at eval (eval at <anonymous> (alpinejs:5:648), <anonymous>:3:32)
    at alpinejs:5:985
    at qt (alpinejs:1:4878)
    at alpinejs:5:27821
    at r (alpinejs:5:15722)
    at Object.Mr [as effect] (alpinejs:5:15510)
    at K (alpinejs:1:375)
    at alpinejs:1:488
    at Function.<anonymous> (alpinejs:5:27814)
    at r (alpinejs:5:1705)
```

This also appears in the warnings
```
Alpine Expression Error: day is not defined

Expression: "day"
<div x-text="day" class="text-gray-800 font-medium text-center text-xs uppercase cursor-default">Sat</div>
```

Code-wise the error appears to be here in the template 
```
        <div class="flex flex-wrap mb-3 -mx-1">
            <template x-for="(day, index) in DAYS" :key="index">
            <div style="width: 14.26%" class="px-0.5">
                <div x-text="day" class="text-gray-800 font-medium text-center text-xs uppercase cursor-default"></div>
            </div>
            </template>
        </div>
```

**Comment by gkimpson** [comment]
Made a pull request - that seems to work :)

**Comment by mkocansey** [comment]
@gkimpson 
Did it work when you removed Alpine Js 2.x from the component and used your project's Alpine 3.x?

**Comment by gkimpson** [comment]
https://github.com/mkocansey/bladewind/pull/18/commits

**Comment by mkocansey** [comment]
@gkimpson Thanks a lot for raising this. Fixed in current release **v0.1.9**
